src/data/finsupp/basic.lean

```diff
@@ -1281,6 +1281,23 @@ lemma multiset_sum_sum [has_zero M] [add_comm_monoid N] {f : α →₀ M} {h : 
   multiset.sum (f.sum h) = f.sum (λa b, multiset.sum (h a b)) :=
 (multiset.sum_add_monoid_hom : multiset N →+ N).map_sum _ f.support
 
+section disjoint_prod_add
+variables [add_comm_monoid M]
+/-- For disjoint `f1` and `f2`, and function `g`, the product of the products of `g`
+over `f1` and `f2` equals the product of `g` over `f1 + f2` -/
+lemma prod_add_index_of_disjoint {f1 f2 : α →₀ M} (hd : disjoint f1.support f2.support)
+  {β : Type*} [comm_monoid β] (g : α → M → β) :
```

Comment:
```suggestion
  {β : Type*} [comm_monoid β] {g : α → M → β} :
```

Reply:
Ah, I had changed this from `{ }` to `( )` in response to @robertylewis 's comment a few days ago.

Reply:
Oh OK!

Let's leave it as it stands then. Perhaps it will help rewrites having one argument explicit.